src/main/java/io/codelex/practice/exercise3.java

```diff
@@ -0,0 +1,64 @@
+package io.codelex.practice;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+class DatePeriod {
```

Comment:
Izlasi vēlreiz uzdevuma nosacījumus, tajos konkrēti ir pateikts:
"The intersection method should receive DatePeriod as a parameter and return DatePeriod", neredzu tādu metodi, kura saņem DatePeriod un atgriež Date period.